CalculatorExercises/1-CalculatorBasic/Main.java

```diff
@@ -0,0 +1,27 @@
+import java.util.*;
+import java.lang.*;
+
+public class Main {
+
+    public static void main(String[] args) {
+        
+        Scanner scan = new Scanner(System.in);
+
+        Calculator calculator = new Calculator();
+        
+        while(true) {
+
+            System.out.println(calculator.getResult());
```

Comment:
In exercise when result is full number like 0,1,2 etc. it still is displayed as 0.0 1.0 2.0. Write method toString() in Calculator that will return number as string like we want it, without redundant .0